app/src/main/java/mozilla/lockbox/view/AppWebPageFragment.kt

```diff
@@ -45,13 +47,25 @@ class AppWebPageFragment : BackableFragment(), WebPageView {
 
         presenter = AppWebPagePresenter(this, url)
 
-        var view = inflater.inflate(R.layout.fragment_webview, container, false)
+        val view = inflater.inflate(R.layout.fragment_webview, container, false)
```

Comment:
😍 

You might also consider using `.also` here.

Reply:
🤔 I've never seen `also` before! _mind blown gif_